src/CMakeLists.txt

```diff
@@ -67,6 +67,13 @@ ELSEIF(NOT ("${SDL_INCLUDE_DIR}" STREQUAL ""))
 	INCLUDE_DIRECTORIES("${SDL_INCLUDE_DIR}")
 ENDIF()
 
+IF(LMMS_HAVE_WEAKJACK)
+	LIST(APPEND LMMS_SRCS "${WEAKJACK_INCLUDE_DIRS}/weak_libjack.c")
```

Comment:
@lukas-w I wasn't really to sure where the best place for these were, so I placed them here.   A second set of eyes is appreciated, but I think it's ready to merge.